### PR TITLE
edit create user

### PR DESCRIPTION
**Pull request by BelehovEgor**
added change office request and change role request

### CODE REVIEW

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 1]
```diff
@@ -94,6 +97,60 @@ private void ChangeUserPosition(Guid positionId, Guid userId, List<string> error
...
+                if (!response.Message.IsSuccess || !response.Message.Body)
+                {
+                    _logger.LogWarning(logMessage, roleId, userId);
+
+                    errors.Add(errorMessage);
+                }
+            }
+            catch (Exception exc)
+            {
+                _logger.LogWarning(exc, logMessage, roleId, userId);
```
Comment (djino97) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:121]: LogError

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 2]
```diff
@@ -94,6 +97,60 @@ private void ChangeUserPosition(Guid positionId, Guid userId, List<string> error
...
+                if (!response.Message.IsSuccess || !response.Message.Body)
+                {
+                    _logger.LogWarning(logMessage, officeId, userId);
+
+                    errors.Add(errorMessage);
+                }
+            }
+            catch (Exception exc)
+            {
+                _logger.LogWarning(exc, logMessage, officeId, userId);
```
Comment (djino97) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:148]: LogError

**Review by djino97 — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/UserService.Business/Commands/User/CreateUserCommand.cs:121]: fixed

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/UserService.Business/Commands/User/CreateUserCommand.cs:148]: fixed

**Review by borovHokage — APPROVED** [review]

**Pull request merged**